src/renderer/api-decorator.js

```diff
@@ -342,6 +342,16 @@ limitations under the License.
         }, 1);
     }
 
+    rqr('electron').remote.getCurrentWebContents(renderFrameId).once('navigation-entry-commited', () => {
```

Comment:
With this addition, `rqr('electron')` appears 5 times in this file now, lets add an `electron` variable at top of file:
```js
    let rqr = require;
    let electron = rqr('electron');
    let webFrame = electron.webFrame.createForRenderFrame(renderFrameId);
    let ipc = electron.ipcRenderer;
    // etc.!
```